styles/style.scss

```diff
@@ -191,10 +195,140 @@ label {
   font-size: 0.8rem;
 }
 
+.modal {
```

Comment:
jako że to jest modal który pojawia się nad resztą treści to możnaby dać ciemne, półprzezroczyste tło wkoło modala.
+ obecnie wygląda to tak że na tel widać dół strony, na ipadzie modal zajmuje cały ekran a na ipad pro wyglada podobnie do desktopa. do przemyślenia: może na mobile ukryć wszystko żeby był tylko modal tak jak na ipad? i tak trzeba scrollować żeby zobaczyć kawałek ktory na który nie można i tak kliknąć

Reply:
miałam plan żeby przy wyświetlaniu game over tło dostawało małe opacity - ale może faktycznie lepiej dać po prostu jakieś tło pod modal... to samo z mobilkami - w tym wypadku w grę wchodzi tylko wyłączenie widoczności głównego layoutu (to spoko pomysł, ale już trzeba będzie obsłużyć). Jeżeli tak będzie ok - wolałabym zmergować to, co jest i w międzyczasie zastanowię się jak to rozwiązać - ale poszłabym w rozwiązanie - do powiedzmy 1024 modal dostaje tło, a poniżej usuwamy layout. Co Ty na to?